ts/components/ChooserListContainer.tsx

```diff
@@ -5,35 +5,47 @@ import { Millisecond } from "italia-ts-commons/lib/units";
 import { debounce } from "lodash";
 import { Body, Button, Content, Input, Item, Right, View } from "native-base";
 import * as React from "react";
+import { ComponentProps } from "react";
 import {
   ImageSourcePropType,
   KeyboardAvoidingView,
-  ListRenderItem,
   Platform,
   StyleSheet
 } from "react-native";
 import variables from "../theme/variables";
 import customVariables from "../theme/variables";
 import ChooserList from "./ChooserList";
+import ChooserListItem from "./ChooserListItem";
 import ChooserListSearch from "./ChooserListSearch";
+import {
+  InjectedWithItemsSelectionProps,
+  withItemsSelection
+} from "./helpers/withItemsSelection";
 import AppHeader from "./ui/AppHeader";
 import FooterWithButtons from "./ui/FooterWithButtons";
 import IconFont from "./ui/IconFont";
 
-type Props<T> = {
+type OwnProps<T> = {
   items: ReadonlyArray<T>;
-  keyExtractor: (item: T, index: number) => string;
-  renderItem: ListRenderItem<T>;
+  initialSelectedItemIds: Option<Set<string>>;
+  keyExtractor: (item: T) => string;
+  itemTitleExtractor: (item: T) => string;
   onCancel: () => void;
+  onSave: (selectedItemIds: Option<Set<string>>) => void;
   isRefreshEnabled: boolean;
   isRefreshing?: boolean;
   onRefresh?: () => void;
   isSearchEnabled: boolean;
```

Comment:
What do you think about removing **isSearchEnabled** prop?
What does it happen if you set **isSearchEnabled : true** and **matchingTextPredicate : undefined**? Does it make sense?
You could say a search is active when you have a predicate set.

Reply:
Yes, I used matchingTextPredicate to enable the search. Done in commit 0c15e67